<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(122, 158, 159);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 245, 219);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(184, 216, 216);</string>
      </property>
      <property name="text">
       <string>SetDeadLine</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Category :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboCategory">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 216, 216);
color: rgb(0, 0, 0);</string>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Study</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Self development</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Workout</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Entertainment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>work</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Freelancing</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Priority :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboPriority">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(184, 216, 216);
color: rgb(0, 0, 0);</string>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High Priority</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medium Priority</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Low Priority</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 216, 216);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>AddTask</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDelete">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 216, 216);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>DeleteTask</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnSave">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 216, 216);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnLoad">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(184, 216, 216);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 245, 219);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
